src/shared/components/Combobox.tsx

```diff
@@ -5,6 +5,11 @@ import CreatableSelect from "react-select/creatable";
 // The React Select is almost impossible to test because they didn't provide any api
 // and the implementation wasn't designed to be testable from outside
 
+export type ComboboxItem = {
+	value: string;
```

Comment:
value should be of type any

Reply:
Np, but could you, please, explain why do you think it'd be better?
 if it should be string only because ReactSelect expects to receive exactly that type.
In case of own component, we could handle any types etc.
![image](https://user-images.githubusercontent.com/9897782/69912059-21b7d100-142d-11ea-95c6-53afe96be810.png)

Reply:
done